### ISSUE TITLE
Feature request: ability to use the selection (move/copy/paste) tool while zoomed in

### ISSUE THREAD

**Issue by sethmnielsen** [ddvk/remarkable-hacks#65]
Hello ddvk,

**TL; DR**: Would it be possible to add the ability to use the selection tool while zoomed in? 

**Explanation**:
I use my rM for working out long engineering problems (i.e. lots of math and derivations), and I often fill up multiple pages while doing so. Flipping back and forth between pages to see all that I've wrote is really inefficient...I can't look at everything all at once like I could if I were using an iPad or OneNote which allow zooming out to get an overview of everything you've wrote.

My solution to this has been zooming in, writing my work using the Fineliner or Mechanical pencil with thickness of 1, then move around the page to continue writing, then zoom out if I need to see the whole problem. But as most people know, zooming in kind of gives you a handicapped experience; moving around the page is very laggy and imprecise with such a small box to work with (it also doesn't allow a real-time preview while dragging it), zooming out then returning back to the level of zoom and the position you were at before is difficult and time consuming, and the worst problem of all: no selection tool! :'( I can't copy/cut and paste (which is super nice to do for long equations) or move the content to make adjustments, which I have to do very often. Zooming out, then using the selection tool, zooming back in, and then getting back to the spot I was at can really throw off one's focus on the problem. Sometimes what I need to select is very small and crunched between a lot of other stuff, and selecting it from the outermost zoom level can be very finicky.

If implementing this is too difficult, then please don't worry about it. Maybe - just maybe - we'll get this feature from the reMarkable team some day. I just hope it's before I graduate in 3 years...

By the way, I absolutely love this project. The reMarkable wouldn't be half of what it is right now without your patches. Thank you for all your incredible work!

**Comment by ddvk** [comment]
i enabled the button, but the behaviour is buggy (seems to work at 200% but not at 500)

**Comment by sethmnielsen** [comment]
Wow, that's interesting. 200% is still great, though! That's the default zoom level and the zoom level I normally work in. 

Out of curiosity, does it work at 400%? Just thinking that 200% doubles the size of everything, and 400% would just be doubling the size a second time.

**Comment by ddvk** [comment]
just try it out, i dont use zoom mode much
